# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/security_level.h
# Commit: 7ba557fab26e
# Full Hash: 7ba557fab26e27e366372eaf3d78997f71185b20
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-07-25 21:49:06
# Description:
#   Bug 1565848: Revert latest change to MITIGATION_DLL_SEARCH_ORDER. r=aklotz
#   
#   This is until any regressions can be fixed, see bug 1568850.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D39357
# ==============================================================================

diff -r d9ac2124b930 -r 7ba557fab26e security/sandbox/chromium/sandbox/win/src/security_level.h
--- a/security/sandbox/chromium/sandbox/win/src/security_level.h	Thu Jul 25 21:26:27 2019 +0300
+++ b/security/sandbox/chromium/sandbox/win/src/security_level.h	Thu Jul 25 17:44:24 2019 +0000
@@ -197,15 +197,10 @@
 // PROCESS_CREATION_MITIGATION_POLICY_STRICT_HANDLE_CHECKS_ALWAYS_ON.
 const MitigationFlags MITIGATION_STRICT_HANDLE_CHECKS = 0x00000100;
 
-// Strengthens the DLL search order. See
-// http://msdn.microsoft.com/en-us/library/windows/desktop/hh310515. In a
-// component build - sets this to LOAD_LIBRARY_SEARCH_DEFAULT_DIRS allowing
-// additional directories to be added via Windows AddDllDirectory() function,
-// but preserving current load order. In a non-component build, all DLLs should
-// be loaded manually, so strenthen to LOAD_LIBRARY_SEARCH_SYSTEM32 |
-// LOAD_LIBRARY_SEARCH_USER_DIRS, removing LOAD_LIBRARY_SEARCH_APPLICATION_DIR,
-// preventing DLLs being implicitly loaded from the application path. Must be
-// enabled after startup.
+// Sets the DLL search order to LOAD_LIBRARY_SEARCH_DEFAULT_DIRS. Additional
+// directories can be added via the Windows AddDllDirectory() function.
+// http://msdn.microsoft.com/en-us/library/windows/desktop/hh310515
+// Must be enabled after startup.
 const MitigationFlags MITIGATION_DLL_SEARCH_ORDER = 0x00000200;
 
 // Changes the mandatory integrity level policy on the current process' token
